addons/web_tour/static/src/js/tip.js

```diff
@@ -175,16 +178,17 @@ var Tip = Widget.extend({
 
         // If the direction or the anchor change : The tip position is updated.
         if (this.info.position !== position || this.viewPortState !== viewPortState) {
+            $('.o_tooltip').removeClass(this.info.position).addClass(position);
```

Comment:
So you search in the whole DOM for tooltips and you change their position... heu?

Reply:
I knew I could do this.$... or something else to not search in the viewport. I just did it to see if it worked and planed to improve it after :).

Reply:
I'm blind, it's obviously this.$el the tooltip with the direction class xD

Reply:
Stop "planning to improve it after". When you push something then ask to review, it should be the final version.